### ISSUE TITLE
Receipt Roll

### ISSUE THREAD

**Issue by Xenomorph99** [Templarian/MaterialDesign#6710]
### Before submitting a new icon contribution, please confirm the following:

- [X] I have [searched all issues](https://github.com/Templarian/MaterialDesign/issues?q=is%3Aissue) to make sure there isn't a request for this icon.
- [X] I have [searched the current library](https://materialdesignicons.com/) to make sure the icon doesn't already exist.
- [X] I understand that [brand icons are no longer being added](http://dev.materialdesignicons.com/roadmap/brand-icons) and I can use [SimpleIcons](https://simpleicons.org/) for my brand icon needs.
- [X] I have followed the [Material guidelines](https://material.io/design/iconography/system-icons.html) for system icons.

### Tell us about your contribution.

I need an icon I can use for a POS receipt printer paper spool. I think calling it `receipt-roll` similar to [paper-roll](https://materialdesignicons.com/icon/paper-roll) makes the most sense, but here's some other options too:

1. `receipt-roll`
2. `receipt-spool`
3. `printer-pos-spool`
4. `printer-pos-paper`

Personally, I like the version that points up the best. I also included a variant that points to the right. It's probably best to include both versions in MDI. That way, the modifiers are properly oriented instead of being in the wrong place if engineers rotate the icon 90º in UI applications.

## Receipt Roll

![receipt-roll](https://user-images.githubusercontent.com/1837101/179613746-5ccf6014-5bbb-432e-910c-46ff608abe75.png)
![receipt-roll-outline](https://user-images.githubusercontent.com/1837101/179613736-775fae3e-0741-4d65-a198-45afbae25ee7.png)
![receipt-roll-off](https://user-images.githubusercontent.com/1837101/179613733-d8ea06e4-6a40-41cc-a01c-1841ea7cf88d.png)
![receipt-roll-off-outline](https://user-images.githubusercontent.com/1837101/179613729-3b6715c0-e200-4385-aaad-1e4a70772fa8.png)
![receipt-roll-alert](https://user-images.githubusercontent.com/1837101/179613728-9017837d-4719-4a7f-877d-283a239e4378.png)
![receipt-roll-alert-outline](https://user-images.githubusercontent.com/1837101/179613723-e1ac4be8-6ec6-4253-9a9c-793d0793448f.png)

## Receipt Roll Variant

![receipt-roll-variant](https://user-images.githubusercontent.com/1837101/179613743-c62c59bb-df67-4f68-a03d-3a85ab9f603c.png)
![receipt-roll-variant-outline](https://user-images.githubusercontent.com/1837101/179613741-2e5eb76c-a00d-45b5-8805-06dc0f067616.png)
![receipt-roll-variant-off](https://user-images.githubusercontent.com/1837101/179613740-bb04709f-19ff-4156-baab-2d7634e5a1d5.png)
![receipt-roll-variant-off-outline](https://user-images.githubusercontent.com/1837101/179613739-d7d13ac5-8ed9-4973-9008-f435f7973880.png)
![receipt-roll-variant-alert](https://user-images.githubusercontent.com/1837101/179613738-06d159bb-922f-4a29-bfe4-27b1d2c8ad1a.png)
![receipt-roll-variant-alert-outline](https://user-images.githubusercontent.com/1837101/179613737-dbae5e57-5b4a-4e37-a9ff-be2012b4b964.png)

### Upload the icon source.

[receipt-roll.zip](https://github.com/Templarian/MaterialDesign/files/9135626/receipt-roll.zip)

**Comment by goyney** [comment]
I really don't see how `paper-roll` doesn't already fit this use case. If anything, this is a variant of that, but the variations are so small (the jagged edge and perforation), that I don't even know if that is needed. Additionally, we've been adamant about not adding new icons that are simple rotates or transforms of existing icons. IMO, this is superfluous.

If we want to add modifiers of the existing base, I can get behind that. But as of now, I don't think this is needed.

**Comment by Templarian** [comment]
Visually this doesn't really work. It might work as a smaller roll focusing more on the receipt part.

![Untitled1](https://user-images.githubusercontent.com/338885/179672268-1774f0b8-4dd4-49e8-afac-98f241fc0bff.png)

**Comment by Xenomorph99** [comment]
Yes, the smaller the part sticking up gets, the more it becomes similar to `paper-roll`. I was trying to avoid the similarity because my app is used in restaurant kitchens. `paper-roll` looks too much like toilet paper. I don't want to people using my app to think about toilet paper while they're in a kitchen context.

![labels](https://user-images.githubusercontent.com/1837101/179793869-4be3b113-4ecc-4c99-8822-99c26f967be4.jpg)
![label-printer](https://user-images.githubusercontent.com/1837101/179793853-6eb68262-d09a-4e64-97c9-85b0e2ca32e1.jpeg)

I think calling this new icon `paper-roll-variant` is a great idea. I'm also fine adding only the non-variant version I designed. I can see if I can come up with some other ideas.

![receipt-roll-outline (1)](https://user-images.githubusercontent.com/1837101/179792246-79b079f4-7604-46cd-a604-29d4be0d4494.png)
![receipt-roll-outline](https://user-images.githubusercontent.com/1837101/179792262-1beea6a8-03a2-4afb-b123-df2414e07e46.png)
